### PR TITLE
Run pre-commit through poe

### LINKED ISSUE #125: black, isort, flake8 should run locally
The pre-commit hook should run locally so that it passes `black`, `isort` and `flake8` before doing a local commit.

It used to be that there was a `poe` command to run `black` and `isort`, but that seems to have disappeared.

### PR DESCRIPTION
**Pull request by twoertwein**
- [x] Closes #125

### CODE REVIEW

File: docs/tests.md [hunk 1]
```diff
@@ -4,12 +4,13 @@
   - Profiles: </br>
     - Default: Runs only mypy and pyright tests </br> `poe test_src --profile=default` </br>
     - Pytest: Runs only pytest </br> `poe test_src --profile=pytest` </br> 
-    - Full: Run all tests (mypy, pyright and pytests) </br> `poe test_src --profiel=full` </br> </br>
+    - Pre-commit: Runs all style checks in pre-commit </br> `poe test_src --profile=style` </br> 
+    - Full: Run all the above tests </br> `poe test_src --profiel=full` </br> </br>
```
Comment (Dr-Irv) [inline docs/tests.md:8]: `profile` not `profiel`

**Review by Dr-Irv — CHANGES_REQUESTED** [review]
One spelling issue.

Is there a way that we can have the pre-commit run as a hook like is done with `pandas` ?

**Comment by twoertwein** [comment]
> Is there a way that we can have the pre-commit run as a hook like is done with `pandas` ?

I will add instructions for it to the docs: only need to run `pre-commit install`.

**Review by Dr-Irv — APPROVED** [review]
thanks @twoertwein

**Pull request merged**